includes/documents/class-wcpdf-invoice.php

```diff
@@ -349,13 +349,13 @@ public function get_pdf_settings_fields( $option_name ) {
 						'30'     => __( '30 days', 'wpo_wcpdf_pro' ),
 						'custom' => __( 'Custom days', 'wpo_wcpdf_pro' ),
 					) ),
-					'description'	=> __( 'Select a due date to display in the invoice.', 'woocommerce-pdf-invoices-packing-slips' ),
+					'description'	=> __( 'Displays a due date below the order data (invoice number, order date, etc.).', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
I think you could avoid the parenthesis content.

Reply:
I was just worried if it was not clear enough.